Bukkit/src/main/java/com/plotsquared/bukkit/util/BukkitUtil.java

```diff
@@ -470,6 +471,12 @@ public void setFoodLevel(final @NonNull PlotPlayer<?> player, @NonNegative final
                 allowedInterfaces.add(Firework.class);
             }
             case "player" -> allowedInterfaces.add(Player.class);
+            case "interaction" -> {
+                final int[] version = PlotSquared.platform().serverVersion();
+                if (version[1] >= 19 && version[2] >= 4) {
```

Comment:
Is the version check really necessary?
Currently, it will not work with 1.20.3.

Reply:
The entity was added in 1.19.4. Maybe its necessary to adjust the version check in order for 1.20.3 to work

Reply:
The `/Bukkit` part of PS is built with 1.20 anyway, as far as I understand the Gradl setup correctly.
https://github.com/IntellectualSites/PlotSquared/blob/5642061d6f848b6d7b4a3ba389caddd9e291b750/Bukkit/build.gradle.kts#L104